vocabularies/DataCite-resourceType.ttl

```diff
@@ -0,0 +1,212 @@
+@prefix qudt:  <http://qudt.org/schema/qudt#> .
+@prefix owl:   <http://www.w3.org/2002/07/owl#> .
+@prefix ssd:   <http://www.w3.org/ns/sparql-service-description#> .
+@prefix rr-entity: <http://registry.it.csiro.au/structure/entity-type/> .
+@prefix xsd:   <http://www.w3.org/2001/XMLSchema#> .
+@prefix skos:  <http://www.w3.org/2004/02/skos/core#> .
+@prefix rdfs:  <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix spec:  <http://www.opengis.net/ont/modspec#> .
+@prefix qb:    <http://purl.org/linked-data/cube#> .
+@prefix dgu:   <http://reference.data.gov.uk/def/reference/> .
+@prefix dct:   <http://purl.org/dc/terms/> .
+@prefix ui:    <http://purl.org/linked-data/registry-ui#> .
+@prefix reg:   <http://purl.org/linked-data/registry#> .
+@prefix api:   <http://purl.org/linked-data/api/vocab#> .
+@prefix vivo:  <http://vivoweb.org/ontology/core#> .
+@prefix vann:  <http://purl.org/vocab/vann/> .
+@prefix prov:  <http://www.w3.org/ns/prov#> .
+@prefix sp:    <http://registry.it.csiro.au/system/prefixes/sp> .
+@prefix foaf:  <http://xmlns.com/foaf/0.1/> .
+@prefix cc:    <http://creativecommons.org/ns#> .
+@prefix op:    <http://environment.data.gov.au/def/op#> .
+@prefix void:  <http://rdfs.org/ns/void#> .
+@prefix dtr:   <http://registry.it.csiro.au/ont/dtr#> .
+@prefix npg:   <http://ns.nature.com/terms/> .
+@prefix odrs:  <http://schema.theodi.org/odrs#> .
+@prefix org:   <http://www.w3.org/ns/org#> .
+@prefix skosxl: <http://www.w3.org/2008/05/skos-xl#> .
+@prefix skos-ext: <http://linked.data.gov.au/def/skos-ext#> .
+@prefix gba:   <http://registry.it.csiro.au/ont/gba-approval/> .
+@prefix version: <http://purl.org/linked-data/version#> .
+@prefix rr-cat: <http://registry.it.csiro.au/structure/category/> .
+@prefix rr-org: <http://registry.it.csiro.au/structure/org/> .
+@prefix rdf:   <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix ldp:   <http://www.w3.org/ns/ldp#> .
+@prefix bibo:  <http://purl.org/ontology/bibo/> .
+@prefix rr-ui: <http://registry.it.csiro.au/structure/ui/> .
+@prefix time:  <http://www.w3.org/2006/time#> .
+@prefix vs:    <http://www.w3.org/2003/06/sw-vocab-status/ns#> .
+@prefix dc:    <http://purl.org/dc/elements/1.1/> .
```

Comment:
```suggestion
@prefix owl:   <http://www.w3.org/2002/07/owl#> .
@prefix xsd:   <http://www.w3.org/2001/XMLSchema#> .
@prefix skos:  <http://www.w3.org/2004/02/skos/core#> .
@prefix rdfs:  <http://www.w3.org/2000/01/rdf-schema#> .
@prefix dct:   <http://purl.org/dc/terms/> .
@prefix reg:   <http://purl.org/linked-data/registry#> .
```

A large number of prefixes not used within the body of the vocabulary. Preferable to remove these, unless there is a known need in the foreseeable future to augment the vocabulary using these additional namespaces.

Reply:
Changes accepted.